v2/azure-arm.yaml

```diff
@@ -478,6 +478,12 @@ status:
 #
 # Available 'type' modifiers:
 #
+# $azureGeneratedSecrets: <array>
+#     The name of secrets generated by Azure which can be automatically
```

Comment:
I found "name of secrets" confusing because the two ends had different plurality; also think we can make this clearer.

```suggestion
# $azureGeneratedSecrets: <array of strings>
#     An  array of names identifying the secrets generated by Azure which can be automatically
```

Reply:
Done, and added a bit more clarification at the end as well.